content/_index.md

```
@@ -5,6 +5,25 @@ title = ""
 
 # Latest Posts
 
+<a href="/../tmp/manuscript/ai/ai-coding-agents-are-blind-to-your-company-knowledge-heres-the-fix"><img src="/../tmp/manuscript/ai/ai-coding-agents-are-blind-to-your-company-knowledge-heres-the-fix/thumbnail.jpg" style="width:50%; float:right; padding: 10px"></a>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)